src/api/article/controller.js

```diff
@@ -22,7 +22,17 @@ export const getAll = ({ query, user }, res, next) => {
   const pageSize = parseInt(query.pageSize) || 10; // eslint-disable-line radix
   const skip = page * pageSize;
   let data;
-  const articlesQuery = { $and: [{ active: true }, queryUnpublished(user)] };
+  const articlesQuery = {
+    $and: [
+      {
+        active: true,
+        // TODO(uladbohdan): investigate how it was saved to db
+        // https://trello.com/c/uLai119m/265-empty-locales-slug-duplication-bug
+        locales: { $exists: true },
```

Comment:
Could you please explain what was the problem and how that fixes it?

Reply:
I did not solve the problem, I just guaranteed that backend will not return articles with empty locales. How they get into the database is task for you, @UladBohdan :)